src/js/components/DataTable/DataTable.js

```diff
@@ -567,7 +567,13 @@ const DataTable = ({
       adjustedData.length > paginationStep &&
       items &&
       items.length ? (
-        <Pagination alignSelf="end" {...paginationProps} />
+        <Pagination
+          alignSelf="end"
+          // flex none needed for Pagination controls to stay in correct
+          // location and prevent overflow outside container
+          style={{ flex: 'none' }}
```

Comment:
Is there any way this can be handled in paginationProps or something other than `style`?

Reply:
Good comment, I changed this to be `flex={false}` to avoid using the style prop.